packages/aws-lambda/layer/bin/publish-layer.sh

```diff
@@ -100,7 +106,6 @@ while read -r region; do
 
 done <<< "$REGIONS"
 
-echo "step 5/5: cleaning up"
+echo "step 7/7: cleaning up"
 rm -rf $TMP_ZIP_DIR
-rm -rf $ZIP_NAME
-
```

Comment:
Just as a reminder for ourselves, this needs to be reverted at some point, shortly before merging this.

I guess it was for troubleshooting/inspecting the resulting zip.